.github/workflows/jacktrip.yml

```diff
@@ -408,42 +411,66 @@ jobs:
           tar -xf qt.tar.xz
           # move qt to a known location
           mv qt-everywhere* $QT_SRC_PATH
-      - name: build static Qt on Linux / macOS
-        if: matrix.static-qt-version && steps.cache-static-qt.outputs.cache-hit != 'true' && runner.os != 'Windows'
+      - name: build static Qt on Linux
+        if: matrix.static-qt-version && steps.cache-static-qt.outputs.cache-hit != 'true' && matrix.name == 'Linux-x64-qmake-gcc-static'
         run: |
+          # Note that other static Linux runners will fail on first run. This is intentional to prevent two runners
+          # from trying to build and update the same cache item. Just re-run the failed jobs after cache is populated.
```

Comment:
I just noticed it and I like it! I was thinking about some solution like this. Thanks!